src/app/(views)/catalogs/[catalogId]/edit/show-playlist.tsx

```
@@ -0,0 +1,221 @@
+                    className="cursor-help underline-offset-1 self-start"
+                    title={item?.description}
+                  >
+                    <h3 className="text-xs">{item?.title}</h3>
+                  </abbr>
+                ) : (
+                  <h3 className="text-xs self-start">{item?.title}</h3>
+                )}
+
+                <div className="flex gap-2 items-center self-start">
+                  <a
+                    target="_blank"
```

Biome (1.9.4)

[error] Avoid using target="_blank" without rel="noreferrer".

Opening external links in new tabs without rel="noreferrer" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noreferrer" attribute.


(lint/a11y/noBlankTarget)